Gordon360/ApiControllers/DirectMessageController.cs

```diff
@@ -84,6 +84,7 @@ public IHttpActionResult GetRooms()
             return Ok(result);
         }
 
+        public string userID;
```

Comment:
Good catch, This was from a solution attempt for saving things that we would send to the backend so that we could do other things with it. We ended up using another solution and forgot to take this out. Thanks!